common/app/views/fragments/basher.scala.html

```diff
@@ -0,0 +1,19 @@
+@(id: String)(implicit request: RequestHeader)
+
+@import views.support.Basher
+
+@if(Basher.fromId(id) != None) {
+    <div class="facia-basher" data-link-name="basher">
+        @conf.Static.css.inline("KeepItInTheGround").map { css =>
```

Comment:
You probably don't want to hardcode this CSS for all bashers. Instead you probably want to map the id parameter to CSS?